src/scm/bitbucket.js

```diff
@@ -0,0 +1,4 @@
+"use strict";
+
+class Bitbucket {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/76f8108a25ffe1858d5d27c3d7dabf83cc8b0291/src/scm/bitbucket.js#L3"></span>‘Bitbucket’ is defined but never used.</td>
    </tr>
  </tbody>
</table>